sdk/identity/identity/CHANGELOG.md

```diff
@@ -1,29 +1,41 @@
 # Release History
 
-## 2.0.0-beta.1 (Unreleased)
+## 2.0.0-beta.1 (2021-03-23)
 
 This release continues with the changes from `1.2.4` and `1.2.4-beta.1`.
 
+### Breaking changes
+
 - The `getToken` methods will now never return `null`. If a token is not available, we will return a rejected promise.
-- `DefaultAzureCredential`'s implementation for browsers was simplified to throw a simple error instead of trying credentials that were already not supported for the browser.
-- Breaking Change: `InteractiveBrowserCredential` for the browser now requires the client ID to be provided.
-- Documentation was added to elaborate on how to configure an AAD application to support `InteractiveBrowserCredential`.
-- Replaced the use of the 'express' module with a Node-native http server, shrinking the resulting identity module considerably
-- Updated `@azure/msal-node-extensions` to [1.0.0-alpha.6](https://www.npmjs.com/package/@azure/msal-node-extensions/v/1.0.0-alpha.6).
+- `InteractiveBrowserCredential` for the browser now requires a client ID to be provided.
+- `InteractiveBrowserCredential` now only has `loginStyle` and `flow` in the optional parameters when the credential is bundled for browsers. This reflects the intended behavior, since the `loginStyle` has no effect on NodeJS.
+- Removed the `postLogoutRedirectUri` from the optional properties of the `InteractiveBrowserCredential`, to align with other languages.
+- Added `@azure/msal-node-extensions` [1.0.0-alpha.6](https://www.npmjs.com/package/@azure/msal-node-extensions/v/1.0.0-alpha.6) as a dependency.
+  - This dependency requires installing `node-gyp`, and on Linux it also requires installing `libsecret`. Read more here: [link](https://github.com/AzureAD/microsoft-authentication-library-for-js/tree/dev/extensions/msal-node-extensions#building).
+- `DefaultAzureCredential`'s implementation for browsers was simplified to throw a simple error instead of trying credentials that were never supported on the browser.
+  - To use Identity in the browser, please use the `InteractiveBrowserCredential` directly.
+
+### New updates and features
+
+- Documentation was added to elaborate on how to configure an AAD application to support `InteractiveBrowserCredential`: [docuemntation link](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/identity/identity/interactive-browser-credential.md).
+- Updated the `@azure/msal-node` dependency to `^1.0.0`.
+- For the `InteractiveBrowserCredential` for node, replaced the use of the `express` module with a native http server for Node, shrinking the resulting identity module considerably.
+- `DeviceCodeCredential` now can receive its optional parameters as a single parameter object.
```

Comment:
Just saw the two constructor overloads we now have:

![image](https://user-images.githubusercontent.com/16890566/112065813-929f6280-8b22-11eb-94c2-5196056bf744.png)


Given that all the parameters in the second one is already part of the `DeviceCodeCredentialOptions`, why have the second overload at all?

Reply:
I'll make an issue.

Reply:
I want to make sure I align with other languages for this method. We'll have an API alignment meeting on Wednesday.

Reply:
Issue: https://github.com/Azure/azure-sdk-for-js/issues/14420

Reply:
API alignment says that all languages support creating the DeviceCodeCredential using optional parameters
- tenantId
- clientId
- userPromptCallback

JS guidelines says that all optional arguments in the signature should be combined in the options bag or you have constructor overloads for each combination

So, unless you are planning to have these overloads for the constructor as well, I don't see why need the one above
- tenantId, options
- clientId, options
- tenantId, clientId, options
- userPromptCallBack, options
- tenantId, userPromptCallback, options
- clientId, userPromptCallback, options
- tenantId, clientId, userPromptCallBack, options

Reply:
I'll add the change as part of this PR then.

Reply:
I added it as part of this PR.